### ISSUE TITLE
ShareX randomly terminates, no error

### ISSUE THREAD

**Issue by HoboBox** [ShareX/ShareX#5473]
This problem has persisted on the same machine (no others) over two separate Windows installs, meaning ShareX has generated a completely new AppData folder and is for all intents and purposes a completely clean install, but yet this problem continues and I'm at my wit's end on how to fix it.
At seemingly random intervals, ShareX will just... terminate. It will record nothing in the Log, nor will it present an error message; it will simply die quietly, and I won't notice until the next time I try to capture something or the next time I mouse over its icon in the tray (which will cause the icon to disappear).
This is what the Log looks like when this event occurs:
![image](https://user-images.githubusercontent.com/35194547/111898372-3980e300-89eb-11eb-8af2-001b395be49e.png)
As you can see, there is a five-hour gap with no error message or any activity. At some point during this period the program simply ceased to be.

Any insight on how to fix this would be _greatly_ appreciated, because it's incredibly frustrating to have to relaunch ShareX with no rhyme or reason.

**Comment by DarkMG1** [comment]
I'm having the same exact error. I've tried the following things.
1. Uninstalling Sharex through Windows
After doing this, issue still persisted.
2. Manually quitting ShareX, then deleting the ShareX data folder located in Documents. Then uninstalling and reinstalling ShareX.
The issue still persists. When can we have this fixed asap?
[ShareX-Log-2021-03-REDACTED.txt](https://github.com/ShareX/ShareX/files/6189866/ShareX-Log-2021-03-REDACTED.txt)

**Comment by AAMBIENTT** [comment]
Bump. still happening in 25'

**Comment by seany** [comment]
Bumping as well. Happening with Windows 11, latest stable shareX version

**Comment by AAMBIENTT** [comment]
No matter what i do, it closes itself if the screen stays off for too long. Not sleep or hibernate as those are disabled. I dont know whats happening but im fining it rather infuriating that this is just been ignored, probably best to look elsewhere atr this point

**Comment by InkyPi** [comment]
Has been happening to me constantly for the last couple of days in Windows 11. I have screenshot linked to one of my mouse buttons and multiple times a day it stops working and I realise I need to reopen the entire app. Real PITA

**Comment by Tslat** [comment]
Can confirm still happening on windows 11, 25h2

Not a single line about anything at all in the log.

Only does it if the screens have been off for long enough (I use the 'turn screen off after X time' power setting). 
It does not require that the computer go to sleep or hibernate.

**Comment by AAMBIENTT** [comment]
@Jaex Would be GREAT if you reopened this. Many people are able to reproduce this. I would like to reinstall SharX but this happens constantly to both me and my room mate. No logs are anything available, nothing in Event viewer either.

**Comment by seany** [comment]
Bumping again, still occurring on 25H2 26200.7171

**Comment by DasDunkel** [comment]
TL;DR: we can't fix what we don't understand/can't reproduce, give us more info. (info list at bottom)

(afaik) none of us nerds that provide support have been able to reproduce this behaviour over the last few years, and if we're unable to reproduce it we're unable to give solutions or fix the code.

Since it's a relatively small group of people experiencing it I'd hazard a guess at it being something to do with your setups, this could be something to do with hardware, like using the same model/brand of laptop, or software, like all of you installing the same piece of software after installing Windows/installing the same modified version of Windows

If we had more info about this issue, we might have a better chance at finding the common denominator

### Info that could be helpful;

@InkyPi since you seem to have had it start recently, did you install any new software/hardware recently? Any major updates? Basically any changes to your system you can remember.

Can you give an estimate for how long it takes? An hour? 2? 3? A day?
Are you running an official Windows install? What version?
What version of ShareX are you installing/using, and where are you downloading from?
Have you tried reproducing the issue in safe mode with only ShareX open? Did it still happen?
When you reinstall ShareX, did you delete/move it's data directory? (`Documents\ShareX`)  (ik DarkMG deleted the folder and it changed nothing, but I'd like to make sure on the latest versions)

**Comment by Tslat** [comment]
Can you give an estimate for how long it takes? An hour? 2? 3? A day?
> Letting the computer run overnight (with the screen turning off after an hour or so) seems to make it happen almost completely consistently, although it does happen at other times as well

Are you running an official Windows install? What version?
> Yes, Windows 11 Pro 26100.7171, licensed. Happens on other versions too

What version of ShareX are you installing/using, and where are you downloading from?
> 18.0.1, online download. Have not tested with store edition personally

**Comment by mikestao** [comment]
Additional info if this will help, crashing randomly (not when I am actively using it).
Application error Event ID 1000

<img width="523" height="468" alt="Image" src="https://github.com/user-attachments/assets/21a5dc46-3287-4815-93e9-c4fa40622a52" />

**Comment by Tslat** [comment]
I _also_ have a crash error - however this appears to be at a different time to when ShareX is crashing overnight, so this might be a separate issue?

I don't have any event entries for overnight crashes

<img width="617" height="251" alt="Image" src="https://github.com/user-attachments/assets/9e684f99-7de0-4b9f-a42e-e1543fc3e43b" />

**Comment by Athenya** [comment]
Also get the same random crashes on version 19.0.2.0 on Windows 10
<img width="493" height="209" alt="Image" src="https://github.com/user-attachments/assets/00d38f82-0228-4739-a38a-a89b77cde1c4" />
